Source/WebKit/UIProcess/API/glib/WebKitWebsiteDataManager.cpp

```diff
@@ -479,6 +490,42 @@ static void webkit_website_data_manager_class_init(WebKitWebsiteDataManagerClass
             nullptr, nullptr,
             FALSE,
             static_cast<GParamFlags>(WEBKIT_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY)));
+
+    /**
+     * WebKitWebsiteDataManager:origin-storage-ratio:
+     *
+     * The ratio which is used to calculate the origin data storage based on disk space.
+     * The origin data storage if it is exceeded, request will fail.
```

Comment:
What request would fail? a storage request? or a load request or what? How does it fail? the load fails? or a console message is shown?